tools/ci/log4j.properties

```diff
@@ -17,6 +17,7 @@
 ################################################################################
 rootLogger.level=INFO
 rootLogger.appenderRef.out.ref=FileAppender
+rootLogger.appenderRef.stdout.ref=ConsoleAppender
```

Comment:
Currently, the test output is collected in artifacts, you can see them here: 

![image](https://user-images.githubusercontent.com/5378924/141421077-ce766d2b-4be5-4ec8-a256-a635f6c49891.png)

Reply:
Thanks, I'll revert the change.